src/Application/Adapters/Laravel/Commands/InsightsCommand.php

```diff
@@ -31,16 +31,17 @@ public function handle(): int
 
         $configPath = ConfigResolver::resolvePath($this->input);
 
-        if (! file_exists($configPath)) {
+        if (!file_exists($configPath)) {
```

Comment:
In our preset coding standard, we put a space after 'not operator' : see https://phpinsights.com/insights/style.html#space-after-not